packages/shared/src/schemas/websockets.ts

```diff
@@ -50,6 +56,11 @@ export const CellUpdatedPayloadSchema = z.object({
   cell: CellSchema,
 });
 
+export const CellAiGeneratedPayloadSchema = z.object({
+  cellId: z.string(),
+  output: z.string(),
```

Comment:
Nit: can we name this `source` to match the name of this property on `cell`? It is the source we're getting back here, right?

Reply:
`source` felt a little wrong to me, it's LLM output, not really a code source.